docs/getting-started/configuration.md

```diff
@@ -0,0 +1,307 @@
+</p>
+
+Take note of the `Client ID` and the `Client Secret`. Open 'app-config.yaml',
+and add your `clientId` and `clientSecret` to this file. It should end up
+looking like this:
+
+```
+auth:
+  # see https://backstage.io/docs/auth/ to learn about auth providers
+  providers:
+    github:
+      development:
+        clientId: YOUR CLIENT ID
+        clientSecret: YOUR CLIENT SECRET
+```
+
+Backstage will re-read the configuration. If there's no errors, that's great! We
+can continue with the last part of the configuration. The next step is needed to
+change the sign-in page, this you actually need to add in the source code.
+
+Open `packages/app/src/App.tsx` and below the last `import` line, add:
+
+```
+import { githubAuthApiRef } from '@backstage/core-plugin-api';
+import { SignInProviderConfig, SignInPage } from '@backstage/core-components';
+
+const githubProvider: SignInProviderConfig = {
+ id: 'github-auth-provider',
+ title: 'GitHub',
+ message: 'Sign in using GitHub',
+ apiRef: githubAuthApiRef,
+};
+```
+
+Search for `const app = createApp({` in this file, and below `apis,` add:
+
+```
+components: {
+   SignInPage: props => (
+     <SignInPage
+       {...props}
+       auto
+       provider={githubProvider}
+     />
+   ),
+ },
+```
+
+That should be it. You can stop your Backstage App. When you start it again and
+go to your Backstage portal in your browser, you should have your login prompt!
+
+To learn more about Authentication in Backstage, there's the following docs you
+could read:
+
+- [Adding Authentication](../auth/)
+- [Adding a new Authentication Provider](../auth/add-auth-provider)
+- [Using authentication and identity](../auth/using-auth)
+- [Using organizational data from GitHub](../integrations/github/org.md)
+
+### Setting up a GitHub Integration
+
+The GitHub integration supports loading catalog entities from GitHub or GitHub
+Enterprise. Entities can be added to static catalog configuration, registered
+with the catalog-import plugin, or discovered from a GitHub organization. Users
+and Groups can also be loaded from an organization. While using GitHub Apps
+might be the best way to set up integrations, for this tutorial you'll use a
+Personal Access Token.
+
+Create your Personal Access Token by opening the
+[the GitHub token creation page](https://github.com/settings/tokens/new). Use a
+name to identify this token and put it in the notes field. Choose a number of
+days for expiration. If you have a hard time picking a number, we suggest to go
+for 7 days, it's a lucky number.
+
+<p align='center'>
+  <img src='../assets/getting-started/gh-oauth.png' alt='Screenshot of the GitHub OAuth creation page'>
+</p>
+
+Set the scope to your likings. For this tutorial, selecting "repo" should be
+enough.
+
+In the `app-config.yaml`, search for `integrations:` and add your token, like we
+did in below example:
+
+```
+integrations:
+  github:
+    - host: github.com
+      token: ghp_urtokendeinfewinfiwebfweb
```

Comment:
![](https://media.giphy.com/media/3knKct3fGqxhK/giphy.gif)

Reply:
H4xxorz in ur config